src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -529,7 +529,7 @@ const UniqueTokenExpandedState = ({
                               registrationDate={
                                 ensData?.registration.registrationDate
                               }
-                              showEditButton={hasEditButton}
+                              showEditButton={isENSActionsEnabled}
```

Comment:
not sure about the naming of this prop, doesn't this modify the ENS duration (not edit the ENS)? maybe `showExtendDuration`?

Reply:
done!